lib/elastic_apm/http_adapters/http_rb_adapter.rb

```diff
@@ -1,37 +1,40 @@
-require_relative './abstract_adapter'
+require 'elastic_apm/http_adapters/abstract_adapter'
 begin
```

Comment:
I like this approach: https://github.com/elastic/apm-agent-ruby/blob/695ff27a9a12c69e22b3c89af0a2ba055912a709/spec/elastic_apm/subscriber_spec.rb#L3-L11

Reply:
Yeah, that's fair. I'd skip the `puts` for this, since we don't want to see a message on every startup if we don't want to use http.rb.